applications/nrf_desktop/src/modules/power_manager.c

```diff
@@ -27,7 +27,7 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_POWER_MANAGER_LOG_LEVEL);
 
 
 #define POWER_DOWN_ERROR_TIMEOUT K_SECONDS(CONFIG_DESKTOP_POWER_MANAGER_ERROR_TIMEOUT)
-#define POWER_DOWN_TIMEOUT_MS	 K_SECONDS(CONFIG_DESKTOP_POWER_MANAGER_TIMEOUT)
+#define POWER_DOWN_TIMEOUT_MS	 CONFIG_DESKTOP_POWER_MANAGER_TIMEOUT * MSEC_PER_SEC
```

Comment:
brackets are missing; also, why changing it?

Reply:
Added brackets. K_SECONDS converts to struct k_timeout_t, but POWER_DOWN_TIMEOUT_MS is used in comparison expressions, so it must remain an integer value.